content/en/docs/reference/rules/rule-fields/rule-fields.md

```diff
@@ -11,3 +11,4 @@ Key | Required | Description | Default
 `warn_evttypes` | no | If set to `false`, Falco suppresses warnings related to a rule not having an event type (more on this [below](#rule-condition-best-practices)). | `true`
 `skip-if-unknown-filter` | no | If set to `true`, if a rule conditions contains a filtercheck, e.g. `fd.some_new_field`, that is not known to this version of Falco, Falco silently accepts the rule but does not execute it; if set to `false`, Falco repots an error and exists when finding an unknown filtercheck. | `false`
 `source` | no | The event source for which this rule should be evaluated. Typical values are `syscall`, `k8s_audit`, or the source advertised by a source [plugin](../plugins). | `syscall`
+`append` | no | When set to `true`, it adds conditions and/or exceptions to a previously defined rule instead of overseeding it. Not used when the goal is just to enable an already existing rule. |
```

Comment:
```suggestion
`append` | no | When set to `true`, it adds conditions and/or exceptions to a previously defined rule instead of overseeding it. Not used when the goal is just to enable an already existing rule.  With lists it appends new elements. |
```